### DISCUSSION TITLE
Are there ways for doing anomaly detection with this library?

### DISCUSSION THREAD

**Discussion by MarkTensenSgt** [alan-turing-institute/sktime#710] [General]
Hi! Really like this library.
Are there already some methods implemented for doing anomaly detection with this?
Cheers

**Comment by aiwalter** [comment]
@MarkTensenSgt I am currently adding an outlier detection algorithm called `HampelFilter` #708 , should be in master soon hopefully. What kind of anomalies would you like to detect else? I would also to appreciate having more algorithms about anomaly detection in `sktime`, would you like to contribute sth?
___
Martin Walter <martin_friedrich.walter@daimler.com>, Mercedes-Benz AG on behalf of Daimler TSS GmbH.
https://github.com/Daimler/daimler-foss/blob/master/LEGAL_IMPRINT.md

**Reply by MarkTensenSgt** [reply]
Good to hear. I'm interested in multivariate anomaly detection, and preferably having the opportunity to test a few algorithms to compare

**Comment by fkiraly** [comment]
There are "implicit" ways by reduction, but not a full module. We were thinking of starting one, once there's a cricital mass of motivated contributors 😃 (see roadmap)

**Reply by MarkTensenSgt** [reply]
Alright! I might be interested to contribute.

**Reply by fkiraly** [reply]
great! options are leading this (for this, you should be experienced contributor already), or waiting for this to appear, sometime this year perhaps, and help implementing methods on the wishlist.
